### PR TITLE
Added the upcoming feature updates to the MSEdge Add-ons roadmap documentation

### PR DESCRIPTION
**Pull request by sonakshimore**
Changes - Partner Center link was changed to the path landing on the Microsoft Edge section of Partner Center

Additions - 5 new feature updates have been added to the documentation.

**Rendered pages for review:**
* [https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/whats-new/released-features?branch=pr-en-us-1803](https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/whats-new/released-features?branch=pr-en-us-1803) 
* [https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/whats-new/roadmap?branch=pr-en-us-1803](https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/whats-new/roadmap?branch=pr-en-us-1803)

### CODE REVIEW

**Comment by sonakshimore** [comment]
I have made re-ordered a few items and updated the correct feature status on the roadmap documentation for review.

**Review by vchapel — APPROVED** [review]
LGTM

**Review by ashishnirvana1 — APPROVED** [review]

File: microsoft-edge/extensions-chromium/whats-new/released-features.md [hunk 1]
```diff
@@ -5,76 +5,85 @@ author: MSEdgeTeam
...
+
+For more information about upcoming updates and feature releases, see [Microsoft Edge Add-ons roadmap](roadmap.md).
 
-For more information about updates and feature releases, see [Microsoft Edge Add-ons roadmap](roadmap.md).
 
 <!-- ====================================================================== -->
 ## Microsoft Edge extensions developer portal
 
-The new edition of the developer portal contains all relevant resources and documentation for Microsoft Edge extensions. Check out the latest changes at [Microsoft Edge Add-ons Developer](https://aka.ms/MSEADeveloper).
+The new edition of the developer portal contains all relevant resources and documentations for Microsoft Edge extensions. You can check [Microsoft Edge Add-ons Developer](https://developer.microsoft.com/en-us/microsoft-edge/extensions/).
```
Comment (Reezaali) [inline microsoft-edge/extensions-chromium/whats-new/released-features.md:23]: can we remove the locale?

Comment (vchapel) [reply microsoft-edge/extensions-chromium/whats-new/released-features.md]: fixed

**Review by ashishnirvana1 — APPROVED** [review]

File: microsoft-edge/extensions-chromium/whats-new/released-features.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (vchapel) [reply microsoft-edge/extensions-chromium/whats-new/released-features.md]: fixed

**Review by vchapel — APPROVED** [review]
LGTM

**Pull request merged**